tests/GBTest.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+namespace cn\tests;
+
+use cn\GB2260;
+
+class GBTest extends \PHPUnit_Framework_TestCase
```

Comment:
The class name should be `GB2260Test`, which is in connection with `GB2260.php`.